lib/util.js

```diff
@@ -38,15 +38,15 @@ const inspectDefaultOptions = Object.seal({
   breakLength: 60
 });
 
-const CIRCULAR_ERROR_MESSAGE = 'Converting circular structure to JSON';
+const CIRCULAR_ERROR_MESSAGE = /Circular/i;
 
 var Debug;
 
 function tryStringify(arg) {
   try {
     return JSON.stringify(arg);
   } catch (err) {
-    if (err.name === 'TypeError' && err.message === CIRCULAR_ERROR_MESSAGE)
+    if (err.name === 'TypeError' && err.message.match(CIRCULAR_ERROR_MESSAGE))
```

Comment:
Yes, usually error message comparison is done in test for which we added `common.engineSpecificMessage` API. See [this](https://github.com/nodejs/node-chakracore/blob/xplat/test/sequential/test-module-loading.js#L77-L79) for example.